alica_engine/src/engine/PlanInterface.cpp

```diff
@@ -0,0 +1,88 @@
+#include "engine/PlanInterface.h"
+#include "engine/model/Task.h"
+#include <assert.h>
+
+namespace alica
+{
+
+using SafeRPPointer = std::pair<const RunningPlan, RunningPlan::ScopedReadLock>;
+
+SafeAssignmentView ThreadSafePlanInterface::agentsInEntryPointOfHigherPlan(const EntryPoint* ep) const
+{
+    assert(ep);
+    if (ep == nullptr) {
+        SafeAssignmentView();
```

Comment:
return SafeAssignmentView() or get rid of the assert

Reply:
No. This is a good assert pattern. Assert on bad input, but allow to continue if the assert is continued or compiled out.

Reply:
Ah, I was forgetting that asserts can be compiled out. This still needs the return :)

Reply:
good catch!